core/templates/dev/head/services/UpgradedServices.ts

```diff
@@ -28,6 +30,7 @@ import { WindowDimensionsService } from './contextual/WindowDimensionsService';
 export class UpgradedServices {
   /* eslint-disable quote-props */
   upgradedServices = {
+    'AlertsService': new AlertsService(new LoggerService()),
```

Comment:
Do we still constructor to have the services? If no, are we expecting this to fail the tests?

Reply:
yes